distributed/comm/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import print_function, division, absolute_import
+
+from .core import connect, listen, CommClosedError
+# Register transports
+from . import tcp
+try:
+    import zmq as _zmq
+except ImportError:
+    pass
+else:
+    # XXX this registers the ZMQ event loop, event if ZMQ is unused...
+    from . import zmq
```

Comment:
I haven't seen any when running a very simple TCP benchmark. At the bottom of each event loop is a simple, relatively cheap, system call (either poll() or epoll()).